src/e2e-node/acp.test.ts

```diff
@@ -125,6 +127,28 @@ describe("An ACP Solid server", () => {
     });
   });
 
+  it("can get and set read access for the public", async () => {
```

Comment:
Should this test also check that actually doing an unauthenticated fetch succeeds ?

Reply:
Excellent suggestion, I added this in https://github.com/inrupt/solid-client-js/pull/1361/commits/c91b02b43f5b2bdf20ce942b0396e348b3aa8062.

Problem is, I don't know if I'm missing something, but the unauthenticated fetch still fails after setting public access.

Do you see something obvious I'm missing or is it something we need to report?

Reply:
Oh, that’s interesting:
https://github.com/inrupt/solid-client-js/runs/4329049297?check_suite_focus=true#step:7:30

Reply:
Ah damn, I hoped this was an issue caused by the ACP implementation, that's why I suggetsed adding this test.